### PR TITLE
Labor camp updates

### PR DESCRIPTION
**Pull request by ghost**
### Do not merge.

First off, some screenshots:

_The new, revamped Labor Camp._
space station 13 2015-10-21 165458

_An updated cache of goods as well as a new one with a wumborian fugu._
space station 13 2015-10-21 165513

Now, onto the finer details.
- The Labor Camp Quarters has been added; this area serves as a living space for prisoners and a place for them to gather equipment and rest between mining trips.
- Safety pickaxes have been replaced with capped pickaxes. These pickaxes mine slower and have only 5 brute damage (punching is generally more effective). You can activate them for a 5% chance of removing the caps after 8 seconds, but only if you don't have much bruising.
- Mining scanners have been added to the equipment racks.
- Safety shovels, being deprecated, have been removed.
- Ore is much more plentiful near the labor camp, and laborers will find much more valuable ore while mining.
- The new standard workers' loadout: laborer's overalls, work boots, and labor backpacks. Labor backpacks (sprites by WalterJe) can only hold three items and slow down the wearer.

(Sprite: bspkrs )
- There is a hidden note under one of the beds in the quarters telling the reader to mine south (the location of loot caches and the ore itself).
- There is a new cache of equipment south of the camp itself. It contains a live wumborian fugu and a unique space suit and helmet. The suit and helmet both cause the wearer to take 30% more brute damage and have obvious names and descriptions. The suit slows down the wearer, and the helmet inflicts severe nearsightedness. The wumborian fugu itself is very hard to defeat with nothing but a capped pickaxe. In addition, the room is surrounded by walls, requiring equipment or gibtonite to destroy.
- The small equipment room with a ushanka and soviet uniform now also has a Syndicate toolbox.
- An escape pod has been added to the Labor Camp. Its floors are brig floors (preventing objective completion) but it still allows prisoners with a way to get out of the Labor Camp if nobody picks them up when the shuttle arrives.

### CODE REVIEW

**Comment by bgobandit** [comment]
can there be a way for this ore to ever make it to science

**Comment by Xhuis** [comment]
@bgobandit, only if Security comes to pick it up, which is the case by default even before these edits.

**Comment by bgobandit** [comment]
no I mean, like, how science gets a "New ores available" message when shit shows up at the cargo machine

**Comment by Xhuis** [comment]
No, that only appears with ore redemption machines. The labor camp doesn't have them.

**Comment by bgobandit** [comment]
yes that is what I am saying, can a notification be added

**Comment by Bawhoppen** [comment]
I doubt anyone will ever use the laborer backpack if theres ANY slowdown.

**Comment by Xhuis** [comment]
Say that to the people who use dufflebags.

**Comment by Xhuis** [comment]
In the latest commit:
* Emergency O2 closets have been scattered around the labor camp
* Fixes an undocumented issue on the main Mining outpost where a single tile had the "Space" area

**Comment by Bawhoppen** [comment]
>Say that to the people who use dufflebags.


THATS BECAUSE DUFFLEBAGS HOLD A BUNCH OF SHIT, AND EVEN THEN THEY'RE RARELY USED

**Comment by Xhuis** [comment]
What I'm trying to say is that people will accept a slowdown for a benefit. On the normal station, backpacks are extremely common and you only get a bit more space for the slowdown. On the labor camp, people may accept a slowdown for three extra storage slots that are surprisingly useful in a place where every item is valuable.

**Comment by Bawhoppen** [comment]
3 slots is nothing for that slowdown.

**Comment by Zilenan91** [comment]
Baw, you're assuming people will be fighting on the labor camp. I can see this happening if sec actually used it more, but if you don't intend on fighting, slowdown is manageable and okay. A piece of armor that does it very well is the mining hardsuit. It's spaceworthy and gives you godlike melee protection, but slows you down in atmos. You can stop the slowdown issue by using a jetpack in space, but otherwise, yeah, you're slow, but you're also almost invulnerable to one of the most common sources of damage in the game.

**Comment by Anonus** [comment]
Does this fix all the unconnected pipes on the mining zlevel?

**Comment by Xhuis** [comment]
@Anonus I'm not aware of any unconnected pipes.

**Comment by Anonus** [comment]
There are, unless someone removed them, unconnected pipes under a number of windows near the mining shuttle dock and unconnected pipes under the racks that have pickaxes and shovels in the labor camp.
Are those fixed with this?

**Comment by Xhuis** [comment]
I haven't heard of them. Thanks for letting me know - I'll look into fixing them.

**Comment by AlexCrimson** [comment]
Remove the spacesuit and toolbox. Sec will never use the labor camp if the prisoners can just escape.

**Comment by Xhuis** [comment]
Security who meta to that degree would likely execute the prisoner anyway on the suspicion of them having an uplink implant or hidden objects regardless. In the coming update, the space suit will be behind metal walls anyway, requiring gibtonite or acquisition of the tools beforehand.

**Comment by Xhuis** [comment]
In the latest commit:
* Removed lots of disconnected pipes on the mining z-level.
* Added an escape pod to the Labor Camp (this is not finished).
* Surrounded the space suit cache with walls, effectively making equipment or gibtonite a prerequisite for breaching the cache.

**Comment by Bawhoppen** [comment]
I really think that there being a suit at all will discourage it's use even more than it already is.

**Comment by PKPenguin321** [comment]
how 2 escape:
0. hope and pray that the emergency o2 closets give you a full set of internals (and optionally, grab a bruise pack). if they do then you're good to proceed
1. meta your way down to the syndie toolbox
2. weld through walls to space suit
3. use 15 brute syndie toolbox to kill wumbo
4. (optional: use your bruise pack to heal up after the fight)
5. profit

how 2 prevent escape (as sec):
1. remove any internals masks from the o2 lockers
2. profit

:+1: for making escape from the gulag possible, while also making it easily preventable.

**Comment by Anonus** [comment]
The gulag is barely even used, wasn't used more after it was made impossible to spacewalk to the syndie suit, and this won't make it used any more.
It's basically a prisoner dumping ground, like permabrig but even more permanent.

**Comment by Xhuis** [comment]
The main reason it doesn't get any use is because there's no incentive to go there. It's not any _fun_, and there's a lot of mapping glitches there in general. That's why I made this PR - to give the labor camp some love and hopefully encourage its use.

**Comment by Bawhoppen** [comment]
The mineral rarity is actually a big change for the gulag, but I still think it won't see much use just due to how security operates. 
Add some abandoned crates.

**Comment by Zilenan91** [comment]
Hey Xhuis can you also make there be a sort of gulag compound between the two buildings? If there's even one space breach it'll depressurize the whole thing and kill everyone, even reaching into the main gulag sec place due to the plastic flaps. You may have to move the ore stuff inside to fix this, to be honest.

**Comment by Xhuis** [comment]
@Bawhoppen I believe abandoned crate spawns are determined like ores, or am I wrong?
@Zilenan91 I can easily add some pressurized bridges, yeah.

**Comment by Bawhoppen** [comment]
@Xhuis I believe there are crate spawners where theres a chance to spawn a crate, but I may be wrong.

**Comment by Xhuis** [comment]
I'll look into it.

**Comment by bgobandit** [comment]
@Xhuis I keep saying, if you want people to go there then add something that notifies science when ores are in, otherwise it is literally useless labor

**Comment by Dorsisdwarf** [comment]
Make a system which takes ores off the shuttle automatically

**Comment by Xhuis** [comment]
Might not be some progress on this for a while, as I'm working on vampires.

**Comment by AlexanderUlanH** [comment]
:-1: for a syndie toolbox on the asteroid with everything you'd need to hack your way to freedom without so much as risking burn damage
But an escape method requiring smart and lucky application of gibtonite sounds perfectly reasonable.

File: code/modules/mining/mine_items.dm [hunk 1]
```diff
@@ -74,6 +75,46 @@
...
+	desc = "A pickaxe capped with rubber softeners, considerably dampening its use a weapon. The caps themselves are fitted extremely tightlu."
+	icon_state = "pickaxe_capped"
+	force = 5 //At the end of the day, it's still a pickaxe
+	digspeed = 50 //Capped off. Realistically, it shouldn't be able to destroy things at all.
+	attack_verb = list("smashed", "slammed", "beaten")
+	var/uncapping = 0 //If the pickaxe is being manipulated
+
+/obj/item/weapon/pickaxe/capped/attack_self(mob/user) //Uncapping!
+	if(!ishuman(user))
+		user << "<span class='warning'>You don't have hands to manipulate [src]!</span>"
```
Comment (Dorsisdwarf) [inline code/modules/mining/mine_items.dm]: Monkies. Also, the only other things that could be picking up a pickaxe and using it somehow are xenos, no?

File: code/modules/mining/mine_items.dm [hunk 2]
```diff
@@ -74,6 +75,46 @@
 /obj/item/weapon/pickaxe/proc/playDigSound()
 	playsound(src, pick(digsound),50,1)
 
+/obj/item/weapon/pickaxe/capped //Used at the labor camp
+	name = "capped pickaxe"
+	desc = "A pickaxe capped with rubber softeners, considerably dampening its use a weapon. The caps themselves are fitted extremely tightlu."
```
Comment (Dorsisdwarf) [inline code/modules/mining/mine_items.dm]: typo

**Comment by Dorsisdwarf** [comment]
Why are the quarters so huge for something which will barely be used?

**Comment by Razharas** [comment]
@Xhuis 
Tell when its ready

**Comment by Xhuis** [comment]
Yep. On a brief hiatus for now but I'll let you know when I ~~FIGURE OUT HOW PODS WORK JESUS FUCK~~ work on this.

**Comment by Xhuis** [comment]
In the interest of closing the number of active PRs I have, this is being closed until I decide to continue it, if ever.

**Pull request closed without merging**